Update more topics to reflect the new  Aiven Console Service settings page (#2428)
aiven/devportal: docs/platform/howto/integrations/access-jmx-metrics-jolokia.rst

Enabling Jolokia integration
------------------------------

Enabling Jolokia Integration
----------------------------

To enable Jolokia integration for a specific service, follow these steps:

> #. Access the `Aiven Console <https://console.aiven.io/>`_ and click on the service for which you want to enable Jolokia integration.

Reviewer: ```suggestion
#. Access the `Aiven Console <https://console.aiven.io/>`_ and click the service for which you want to enable Jolokia integration.
```

#. On the **Overview** page of your service, click **Integrations** in the sidebar.

> #. On the **Integrations** page, find **Jolokia** under **External Integrations** and click on it.

Reviewer: ```suggestion
#. On the **Integrations** page, in**External Integrations**, click **Jolokia**.
```

> #. Choose the Jolokia endpoint you created and click **Enable**. The system will then configure the Jolokia endpoint on all service nodes, providing access to the metrics.

Reviewer: ```suggestion
#. Choose the Jolokia endpoint you created and click **Enable**. The system configures the Jolokia endpoint on all service nodes, providing access to the metrics.
```
